src/improvements/tasks/sep/005-opcm-upgrade-v300/VALIDATION.md

```diff
@@ -0,0 +1,402 @@
+```
+
+In mainnet runbooks, this calldata should appear in [Action Plan](https://gov.optimism.io/t/upgrade-proposal-14-isthmus-l1-contracts-mt-cannon/9796#p-43948-action-plan-9) section of the Governance proposal.
+
+# State Changes
+
+## Nested Safe State Overrides and Changes
+
+This task is executed by the nested 2/2 `ProxyAdminOwner` Safe. Refer to the
+[generic nested Safe execution validation document](../../../../../NESTED-VALIDATION.md)
+for the expected state overrides and changes.
+
+The `approvedHashes` mapping of the `ProxyAdminOwner` should change during the simulation.
+See the ["Key Computation" section](../../../../../NESTED-VALIDATION.md#key-computation) in the nested validation doc
+for instruction on how to validate this change.  You can find the target safe hash in the simulation output
+under "Nested Multisig Child's Hash to Approve", which should be:
+- `0x7ae8800316ad25e257c77b54d034e5befc61dc38da97801125cc99f404be529e`
+
+Additionally, Safe-related nonces [will increment by one](../../../../../NESTED-VALIDATION.md#nonce-increments).
+
+## Other Nonces
+In addition to the Safe-related nonces mentioned [previously](#nested-safe-state-overrides-and-changes),
+new contracts will also have a nonce value increment from 0 to 1.
+This due to [EIP-161](https://eips.ethereum.org/EIPS/eip-161) which activated in 2016.
+
+This affects the newly deployed dispute games mentioned in ["State Diffs"](#state-diffs).
+
+## State Diffs
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+### [`0x034edd2a225f7f429a63e0f1d2084b9e0a93b538` ](https://github.com/ethereum-optimism/superchain-registry/blob/1a5d7a208cea9b0ea175df1fe71bdc4da7f4c04c/superchain/configs/sepolia/op.toml#L5) (SystemConfig - OP Sepolia Testnet) Chain ID: 11155420
+
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x760C48C62A85045A6B69f07F4a9f22868659CbCc`
+  - **After:** [`0x340f923E5c7cbB2171146f64169EC9d5a9FfE647`](/Users/copypaste/Optimism/superchain-ops/src/improvements/tasks/sep/005-opcm-upgrade-v300/VALIDATION.md)
```

Comment:
This isn't a proper link. Can you permalink to the actual address in the superchain registry.

Reply:
Contains the proper link now